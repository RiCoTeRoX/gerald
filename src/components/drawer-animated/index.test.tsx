import { render } from '@testing-library/react-native'
import React from 'react'
import { Text, View } from 'react-native'

import DrawerAnimated from '.'

describe('DrawerAnimated', () => {
  it('renders correctly', () => {
    const { toJSON } = render(
      <DrawerAnimated>
        <View>
          <Text>Test</Text>
        </View>
      </DrawerAnimated>
    )
    expect(toJSON()).toMatchSnapshot()
  })
})
