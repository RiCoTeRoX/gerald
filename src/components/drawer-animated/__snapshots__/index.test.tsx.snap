// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DrawerAnimated renders correctly 1`] = `
<View
  style={
    [
      {
        "flex": 1,
      },
      {
        "transform": [
          {
            "translateY": undefined,
          },
        ],
      },
    ]
  }
>
  <View>
    <Text>
      Test
    </Text>
  </View>
</View>
`;
